apps/bfd-server/bfd-server-war/src/test/resources/endpoint-responses/v2/metadata.json

```diff
@@ -46,17 +104,17 @@
         "code" : "search-type"
       } ],
       "searchParam" : [ {
-        "name" : "patient",
-        "type" : "reference",
-        "documentation" : "The patient identifier to search for"
-      }, {
         "name" : "_lastUpdated",
         "type" : "date",
         "documentation" : "Include resources last updated in the given range"
       }, {
         "name" : "excludeSAMHSA",
         "type" : "string",
         "documentation" : "If true, exclude all SAMHSA-related resources"
+      }, {
```

Comment:
why did the searchParam order change for Coverage?

Reply:
The order the search params came back in changed when two new resources were added, not sure why or how, guessing because a hashtable is being used somewhere.  There appears to be an attempt to sort the parameters, but it seems they are still coming back in a non-deterministic order, for the sake of this PR I just set the order to what it now currently wants, as the resources themselves, but might do well to fix this later.